cogs/PlaytesterPing.py

```diff
@@ -15,17 +15,10 @@ def getLatestDiscussion():
     query = """
     query {
         repository(owner: "R2Northstar", name: "Northstar") {
-            discussions(first: 1) {
+            discussions(first: 1 categoryId: "DIC_kwDOGkM8Yc4CN-04") {
```

Comment:
Love the only checking for the specific category, eliminates chance of getting the wrong discussion which was a concern